libs/blocks/accordion/accordion.js

```diff
@@ -19,11 +53,20 @@ function handleClick(el, dd) {
     el.setAttribute('aria-expanded', 'true');
     dd.removeAttribute('hidden');
   }
+
+  if (el.closest('.editorial')) {
```

Comment:
```suggestion
 const closestEditorial = el.closest('.editorial');
  if (closestEditorial) displayMedia(closestEditorial.querySelector('.accordion-media'), el, dd, num, id, expanded);
```
note: you're also missing a semicolon